content.js

```diff
@@ -97,22 +104,19 @@ function timeOver() {
     let hostname = window.location.hostname.replace("www.", "");
     let domainName = hostname.split('.')[0];
     domainName = domainName.charAt(0).toUpperCase() + domainName.slice(1);
-    
+
     let continueButton = document.createElement("button");
     continueButton.innerHTML = `Continue to ${domainName}`;
     continueButton.id = "continueButton";
     continueButton.addEventListener("click", function() {
         document.getElementById("overlayElement").style.display = "none";
     });
 
-    // To add newline between buttons
-    let p = document.createElement("p");
```

Comment:
@rezaarkan Since `p`'s definition was removed here, line 115 in the new file is referencing an undefined `p`. Could you please look into this?

Reply:
@rezaarkan I'm surprised the buttons are appearing on your end as my browser did not like an undefined `p` trying to be rendered. Would you be able to double check that the extension on your browser reflects the same code that you have in this PR? Thanks again.

Reply:
@qu8n oh facepalm! I have this change locally and thought that it would be part of the commit already, but I guess I missed that. Thanks for pointing that out!

Reply:
No worries! Thanks for checking 🙂